.gitignore

```diff
@@ -104,4 +104,10 @@ ENV/
 .#*
 
 # Auxilliary files associated with vim
-*.swp
\ No newline at end of file
+*.swp
+
+# Miscellaneous files
+.idea/
+Untitled*
+untitled*
+C\:*
```

Comment:
For some reason on my workstation running CentOS 6.8 I occasionally get files that start with `C:\\nppdf32Log\\debuglog.txt` for some unbeknownst reason.  Hm...maybe `*debuglog*` would be a better choice, in case this has any adverse impact on Windows.

Reply:
Huh, that's weird.

Here's something I found out about recently that might be helpful, though: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore